src/main/java/org/smartregister/pathevaluator/dao/TaskDao.java

```diff
@@ -36,6 +36,6 @@
 
 	org.smartregister.domain.Task getTaskByIdentifier(String id);
 
-	void updateTask(org.smartregister.domain.Task task);
+	boolean updateTask(org.smartregister.domain.Task task);
```

Comment:
Maybe we may want to return the updated task, e.g we we want to use the new updated task to refresh UI